### ISSUE TITLE
acc and loss remains the same

### ISSUE THREAD

**Issue by teddyjkwang** [daviddao/spatial-transformer-tensorflow#3]
THX for your great effort.
I try to run the code but found that acc remains 0.119 for a long time and loss remains about 2.3000. How to optimize it?

**Comment by xyyu-callen** [comment]
@daviddao 
Many thanks.
I have got the same problem. 
The accuracy is always 0.119 and the loss is about 2.3000.
Can you help us to solve it?

**Comment by daviddao** [comment]
Try to restart the training and tune hyperparameters. One case why the accuracy and loss get stuck might be because the transformer zoomed `in` too radical into an empty part of the image!

**Comment by chl916185** [comment]
@SwaggyK93 @xyyu-kevin 
Accuracy (498): 0.957
Iteration: 0 Loss: 0.0012747037
Iteration: 10 Loss: 0.0026973477
Iteration: 20 Loss: 0.0013430312
Iteration: 30 Loss: 0.005542839
Iteration: 40 Loss: 0.00043637512
Iteration: 50 Loss: 0.0073211356
Iteration: 60 Loss: 0.0051542623
Iteration: 70 Loss: 0.007538586
Iteration: 80 Loss: 0.00058482925
Iteration: 90 Loss: 0.00042093257
Accuracy (499): 0.957
 i try to run the code ,result is 0.957
